PubHub/PubHub.AdminPortal/Components/Auth/SignInOrOut.razor

```diff
@@ -24,6 +26,14 @@
 
     async Task SignOut()
     {
+        var result = await AuthenticationService.RevokeTokenAsync();
+
+        if (!result.IsSuccess)
+        {
+            NotificationService.Notify(new Radzen.NotificationMessage { Severity = Radzen.NotificationSeverity.Error, Summary = result.ErrorDescriptor, Duration = 5000 });
```

Comment:
Are you pushing the errors into the GUI of the client?

Reply:
Ja? Der står bare i den der at de ikke har adgang til sitet..? Det må der måske ikke?

Reply:
Kan snildt lave dem om til custom ones.

Reply:
> Ja? Der står bare i den der at de ikke har adgang til sitet..? Det må der måske ikke?

Jo. Men du kan ikke sikrer at der ikke kommer sensitive data fra API (:
Det er bare en ekstra sikring fordi API errors er tiltænkt devs og ikke end users👍 

Det er bedre at give brugern en generic fejlbesked og inkludere en error code (_Ligesom jeg har gjort i Mobile_). PÅ dne måde kan vi fejlsøge problemet og samtidigt sikrer at senstive data ikke bliver psuhed (:

Reply:
Should be changed.

Reply:
> > Ja? Der står bare i den der at de ikke har adgang til sitet..? Det må der måske ikke?
> 
> Jo. Men du kan ikke sikrer at der ikke kommer sensitive data fra API (: Det er bare en ekstra sikring fordi API errors er tiltænkt devs og ikke end users👍
> 
> Det er bedre at give brugern en generic fejlbesked og inkludere en error code (_Ligesom jeg har gjort i Mobile_). PÅ dne måde kan vi fejlsøge problemet og samtidigt sikrer at senstive data ikke bliver psuhed (:

Reply:
Added it the error codes to some of the places where it made sense - didn't see this when i made the changes. My bag